packages/dds/sequence/README.md

```diff
@@ -1,19 +1,29 @@
 # @fluidframework/sequence
 
-Every item in a SharedSegmentSequence is at a specific position starting at 0, kind of like an array. However, it differs from an array in that the positions can move as local and remote collaborators make modifications to the sequence. There are a number of different sequence types:
-- SharedString for storing and collaborating on a sequence of text
-- SharedNumberSequence for storing and collaborating on a sequence of numbers
-- SharedObjectSequence for storing and collaborating on a sequence of json serializable objects
+Every item in a **SharedSegmentSequence** is at a specific position starting at 0, kind of like an array. However, it
+differs from an array in that the positions can move as local and remote collaborators make modifications to the
+sequence. There are a number of different sequence types:
 
-As the name suggests SharedSegmentSequence, or sequence for short, are made of segments. Segments are the leaf nodes of the tree data structure that enables collaboration and backs the sequence. Segments may be split and merged as modifications are made to the sequence. Every segment has a length from 1, to the length of the sequence. The length of the sequence will be the combined length of all the segments.
+- SharedString for storing and collaborating on a sequence of text.
+- SharedNumberSequence for storing and collaborating on a sequence of numbers.
+- SharedObjectSequence for storing and collaborating on a sequence of JSON-serializable objects.
 
-When talking about positions in a sequence we use the terms _near_, and _far_. The nearest position in a sequence is 0, and the farthest position is its length. When comparing two positions the nearer position is closer to 0, and the farther position is closer to the length.
+As the name suggests SharedSegmentSequence, or sequence for short, are made of segments. Segments are the leaf nodes of
+the tree data structure that enables collaboration and backs the sequence. Segments may be split and merged as
+modifications are made to the sequence. Every segment has a length from 1, to the length of the sequence. The length of
```

Comment:
We're using "length of the sequence" in many places the way a variable is used in code. But it takes the reader a while to figure this out. As an alterative, I suggest using "n (where n is the length of the sequence)". In this case:

```suggestion
modifications are made to the sequence. Every segment has a length from 1 to *n* (where *n* is the length of the sequence). The length of
```
An alternative, but not as good would be to hyphenate so readers understand that it is really one word: "from 1 to length-of-the-sequence".